File: CUSTOR.EICOnline.DAL/EntityLayer/ServiceTariff.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace CUSTOR.EICOnline.DAL
{
  public partial class ServiceTariff
  {
    public ServiceTariff()
    {
    }

    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int? ServiceTariffId { get; set; }

    public int ServiceId { get; set; }
    public int TariffId { get; set; }
    public Tariff Tariff { get; set; }
  }

  public partial class ServiceTariffLookup
  {
    public ServiceTariffLookup()
    {
    }

    public int? ServiceTariffId { get; set; }
    public int ServiceId { get; set; }

    //public int TariffId { get; set; }
    public string ServiceNameEnglish { get; set; }

    public string NameEnglish { get; set; }
    public double? Fee { get; set; }
    public int[] TariffId { get; set; }
  }
}

File: CUSTOR.EICOnline.Angular/src/app/model/servicetariff.ts
import {TariffModel} from './tariff';

export class ServiceTariffModel {
  public ServiceTariffId?: number;
  public ServiceId: number;
  public TariffId: number[];
  public Fee?: number;
  public NameEnglish?: string;
  public Tariff?: TariffModel;

  constructor(serviceTariffId?: number, serviceId?: number, TariffId?: number[], NameEnglish?: string, fee?: number, TariffIdName?: TariffModel) {
    this.ServiceTariffId = serviceTariffId;
    this.ServiceId = serviceId;
    this.TariffId = TariffId;
    this.NameEnglish = NameEnglish;
    this.Fee = fee;
    //this.Tariff = TariffIdName;
  }
}
export class ServiceTariffViewModel {
  public ServiceTariffId?: number;
  public ServiceId: number;
  public TariffId: number;
  public Fee?: number;
  public NameEnglish?: string;
  public Tariff?: TariffModel;
}